sphinx_tabs/static/doctools.js

```diff
@@ -0,0 +1,316 @@
+/*
```

Comment:
I'm a little tentative on overriding core aspects of sphinx 😬 (in case it is changed in some way upstream) are the modifications actually specific to sphinx-tabs or could you make a PR to sphinx to add these modifications?
At the least could you add a link to where the original document can be viewed (on https://github.com/sphinx-doc/sphinx) and clearly comment where the modifications are and why they have been made.

Reply:
Some pretty short research shows that it's near identical to https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/themes/basic/static/doctools.js

Reply:
Actually, it looks like they are identical, not sure why it's overriding?

Reply:
I'd been using this fork, so it was meant as a short term workaround really. But sphinx-doc/sphinx#8316 included the minor change in sphinx, so I'll drop it here 👍

Reply:
Removed this file from the PR